src/lib/Scenes/ArtworkAttributionClassFAQ/ArtworkAttributionClassFAQ.tsx

```diff
@@ -39,7 +39,7 @@ export class ArtworkAttributionClassFAQ extends React.Component<Props> {
     return (
       <Theme>
         <ScrollView>
-          <Box pt={this.props.safeAreaInsets.top} pb={this.props.safeAreaInsets.top} px={2}>
+          <Box pt={this.props.safeAreaInsets.top + 3} pb={this.props.safeAreaInsets.top + 4} px={2}>
```

Comment:
I don't think that manually adding padding here is the right approach. The other components in Emission already avoid the padding somehow, I believe by using a flex container. There are two approaches I can find: 

Adding a containing flex wrapper:

https://github.com/artsy/emission/blob/b65d5727e24619d2c42e43495f9c8e2149a51140/src/lib/Containers/Conversation.tsx#L24-L26

Or modifying the props of the outermost container:

https://github.com/artsy/emission/blob/b65d5727e24619d2c42e43495f9c8e2149a51140/src/lib/Components/Consignments/Screens/Welcome.tsx#L97

I think the second approach would work here, but I haven't tested it out yet 🙈